### PR TITLE
Savingfeature

### PR DESCRIPTION
**Pull request by Harish110**
Pull request build highway

### CODE REVIEW

File: SimpCityTests/GameTests.cs [hunk 1]
```diff
@@ -2,18 +2,21 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using SimpCity;
 
-namespace SimpCityTests {
+namespace SimpCityTests
+{
```
Comment (CaeNGnp) [inline SimpCityTests/GameTests.cs]: Please run the formatter again after merging the `main` branch as cecce1c211e5fec5e19935c3a0d31b5d80928e6f updates the format spec.

File: SimpCityTests/GameTests.cs [hunk 2]
```diff
@@ -108,8 +113,34 @@ public class GameTests {
...
+                {
+                    colcount += 1;
+                    string col = column.ToString();
+
+                }
+                colcount = 0;
+                Console.WriteLine(line);
+            }
+
+        }
```
Comment (CaeNGnp) [inline SimpCityTests/GameTests.cs:135]: Good attempt for making the test script, but this virtually tests nothing with no assertion whatsoever

**Review by CaeNGnp — CHANGES_REQUESTED** [review]
- Please revert "House" related commits, see #40 
- Please merge the latest `main` refs to avoid merge conflicts

**Review by JSQJ — APPROVED** [review]
Testing no issues

**Review by kenwongz — CHANGES_REQUESTED** [review]
@Harish110 please fix the conflict. Please do it by 21/01/22

**Comment by kenwongz** [comment]
@Harish110 pls take a look at the code again to satisfy the requirement.
image

**Comment by kenwongz** [comment]
@Harish110 Please double check your code again
image

File: SimpCity/Game.cs [hunk 3]
```diff
@@ -43,20 +44,43 @@ public class Game {
...
+                       MakeNew = () => new Monument(buildingInfo[BuildingTypes.Monument])
+                   }
+                },
+                {
+                   BuildingTypes.House, new BuildingInfo() {
+                       Code = House.Code,
+                       Name = House.Name,
+                       MakeNew = () => new House(buildingInfo[BuildingTypes.House])
+                   }
+                }
```
Comment (CaeNGnp) [inline SimpCity/Game.cs]: Dirty diff, please put this portion up back where it was above...

File: SimpCity/SimpCity.csproj [hunk 4]
```diff
@@ -47,6 +47,10 @@
     <Compile Include="buildings\House.cs" />
     <Compile Include="buildings\Beach.cs" />
     <Compile Include="buildings\Factory.cs" />
+    <Compile Include="buildings\Highway.cs" />
+    <Compile Include="buildings\House.cs" />
```
Comment (CaeNGnp) [inline SimpCity/SimpCity.csproj]: Duplicate of line 47.

**Review by JSQJ — APPROVED** [review]
Tested, previous issue fixed

**Review by CaeNGnp — CHANGES_REQUESTED** [review]
Still a merge blocker for me. CI Test build is [failing](https://github.com/kenwongz/SimpCity/runs/4867871277?check_suite_focus=true) and formatting issues unresolved.

**Review by CaeNGnp — APPROVED** [review]
That's better

**Comment by Harish110** [comment]
> @Harish110 please fix the conflict. Please do it by 21/01/22

Conflict solved. - harish

**Comment by kenwongz** [comment]
Open this issue again as we have not tested it yet

**Review by JSQJ — APPROVED** [review]
Tested, no issues

**Review by kenwongz — APPROVED** [review]

**Comment by kenwongz** [comment]
Tested by Kenneth and Jeremy

**Pull request merged**